### PR TITLE
fix(docs): refactor sidebar

### PR DESCRIPTION
**Pull request by dhmlau**
Originated from https://github.com/strongloop/loopback-next/pull/3164

In this PR:
- refactor the sidebar as proposed in https://github.com/strongloop/loopback-next/pull/3164
- new files for the top level sidebar items:
  - Access-databases.md (include image)
  - Deployment.md
  - Integrate-with-infrastructure.md
  - Usage-scenarios.md

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/sidebars/lb4_sidebar.yml [hunk 1]
```diff
@@ -137,35 +137,63 @@ children:
...
-  url: How-tos.html
+- title: 'Usage Scenarios'
+  url: Usage-scenarios.html
   output: 'web, pdf'
   children:
 
-    - title: 'Exposing GraphQL APIs'
-      url: exposing-graphql-apis.html
-      output: 'web, pdf'
+  - title: 'Create Other APIs'
```
Comment (raymondfeng) [inline docs/site/sidebars/lb4_sidebar.yml]: `Create Other Forms of APIs`?

File: docs/site/sidebars/lb4_sidebar.yml [hunk 2]
```diff
@@ -191,10 +219,6 @@ children:
     url: Boot-and-Mount-a-LoopBack-3-application.html
     output: 'web, pdf'
 
-- title: 'Calling other APIs'
-  url: Calling-other-APIs-and-web-services.html
-  output: 'web, pdf'
-
```
Comment (raymondfeng) [inline docs/site/sidebars/lb4_sidebar.yml]: There is an extra empty line at https://github.com/strongloop/loopback-next/blob/b7cd2c2bb15b132ca3899d5965519370e7c71a30/docs/site/sidebars/lb4_sidebar.yml#L22

File: docs/site/Integrate-with-infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+lang: en
+title: 'Integrate with Other Infrastructure'
```
Comment (raymondfeng) [inline docs/site/Integrate-with-infrastructure.md]: `Integrate with Other Infrastructure` -> `Integrate with Infrastructure`?

**Comment by dhmlau** [comment]
This is what the new sidebar looks like after the change:

new-sidebar

**Comment by dhmlau** [comment]
Note: the pages for the higher level items are quite empty, but we can add more content over time. Pages include:
- Usage Scenario
- Create Other Forms of APIs
- Access Databases

File: docs/site/Integrate-with-infrastructure.md (hunk 3 above)
Comment (nabdelgadir) [inline docs/site/Integrate-with-infrastructure.md]: `Infrastructures`?

File: docs/site/Integrate-with-infrastructure.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+keywords: LoopBack 4.0, LoopBack-Next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Integrate-with-infrastructure.html
+---
+
+It is easy to integrate your LoopBack application with other infrastructure
```
Comment (nabdelgadir) [inline docs/site/Integrate-with-infrastructure.md]: ditto

File: docs/site/Integrate-with-infrastructure.md (hunk 3 above)
Comment (dhmlau) [reply docs/site/Integrate-with-infrastructure.md]: Fixed. Actually on a  previous comment from @raymondfeng , it should be `Integrate with Infrastructures`. :) Thanks.

**Pull request merged**